src/index.css

```diff
@@ -25,3 +27,218 @@ code {
+  letter-spacing: normal;
+  color: var(--azure);
+  font-family: Montserrat, sans-serif;
+}
+
+.nav-ul {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  gap: 5em;
+}
+
+.head {
+  margin-left: auto;
+  padding: 5%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.book-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 3em;
+}
+
+.book-content {
+  width: 50%;
+}
+
+.Lesson-Panel {
+  display: flex;
+  justify-content: space-between;
+  padding: 2%;
+  border-radius: 4px;
+  border: solid 1px #fff;
+  background-color: #fff;
+}
+
+.lesson-side {
+  display: flex;
+  justify-content: space-evenly;
+  width: 50%;
+}
+
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.title {
+  width: 70%;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1.2em;
+  font-weight: bold;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  padding: 2% 0;
+}
+
+.book {
+  width: 6.688rem;
+  height: 1.188rem;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.progress {
+  display: flex;
+  align-items: center;
+  gap: 2em;
+}
+
+.progress h1 {
+  font-family: Montserrat, sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progress span {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.chapter p {
+  opacity: 0.5;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter h5 {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.9rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+hr {
+  height: 4.375rem;
+  margin-top: 7%;
+}
+
+form {
+  background-color: #f3f3f3;
+  width: 97%;
+  height: auto;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 3%;
+}
+
+.form-h1 {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  padding-left: 3%;
+}
```

Comment:
- [ ] As required by the previous code reviewer, please 🙏🏽 align the form to the rest of the content to look similar to the Zeplin design.

> Kindly align the book form with the book cards above. Currently, your form is a bit to the left. ✍️